docs/schema.rst

```diff
@@ -0,0 +1,50 @@
+Schema
+======
+
+The ``graphene.Schema`` object describes your data model, and provides a GraphQL server with an associated set of resolve methods that know how to fetch data. The most basic schema you can create looks like this:
+
+.. code:: python
+
+    import graphene
+
+    class Query(graphene.ObjectType):
+        pass
+
+    class Mutation(graphene.ObjectType):
+        pass
+
+    schema = graphene.Schema(query=Query, mutation=Mutation)
+
+
+This schema does nothing like this, but it is ready to accept new Query or Mutation fields that you create.
```

Comment:
"does nothing like this" is awkward phrasing and is better suited for a situation where you've been talking about "this" and want to draw a comparison. I might do:

"This schema doesn't do anything yet, but it is ready to accept new Query and Mutation fields."